CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## **3.0.0** <sub><sup>2022-07-22 (4ccba12...e47bbc2)</sup></sub>
+
+### Features
+
+- implement EditableLable component ([4ccba12](https://github.com/Cosmo-Tech/webapp-component-ui/commit/4ccba12))
```

Comment:
Could you fix the typo "EditableLable" in the release notes please ? :D Thank you !